OptimizelySDK.Tests/OptimizelyTest.cs

```diff
@@ -1495,5 +1502,234 @@ public void TestIsFeatureEnabledGivenFeatureFlagIsEnabledAndUserIsBeingExperimen
         }
 
         #endregion // Test IsFeatureEnabled method
+
```

Comment:
ahhh for now am tired. will check it after sometime. 
@oakbani  please my comments and in ruby-sdk review, you consider all these comments and ofcourse in php as well. :)
ajh asal main nind puri hogai thi. :P

Reply:
Aur mujhe aaj neend hi nahi arahi thi, aur koi movie bhi nahi thi queue main. To socha k kuch aisa kara jaey k jis se neend asani se ajaey :P